### ISSUE TITLE
Bup from PPA not working

### ISSUE THREAD

**Issue by Uatschitchun** [ulikoehler/deb-buildscripts#2]
Hi. As I've found no other way to contact you...

Installed bup from you PPA (20.04 focal) and its not working! This is what I get:
```
$ bup 
Traceback (most recent call last):
  File "/usr/bin/bup", line 26, in <module>
    from bup import compat, path, helpers
  File "/usr/lib/bup/cmd/../bup/helpers.py", line 13, in <module>
    from bup import _helpers
ImportError: /usr/lib/bup/cmd/../bup/_helpers.so: undefined symbol: PyBytes_FromStringAndSize
```
When compiling bup from git its working fine. Could you check for a wrong or missing dependency? I've tried installing all dependencies given on bup's github page to no avail. So something's wrong on you PPA version.

**Comment by ulikoehler** [comment]
Hi ! Thanks for reporting this. I'll try to reproduce using a fresh installation of Ubuntu.

**Comment by ulikoehler** [comment]
I've released a new bup v0.32 in the PPA and verified that on a fresh Ubuntu x86_64 installation, when just:
```
sudo add-apt-repository ppa:ulikoehler/bup
apt -y install bup
```

it works:
```
root@ubuntu-s-1vcpu-1gb-fra1-01:~# bup
Usage: bup [-?|--help] [-d BUP_DIR] [--debug] [--profile] <command> [options...]

Common commands:
    fsck       Check backup sets for damage and add redundancy information
    ftp        Browse backup sets using an ftp-like client
    fuse       Mount your backup sets as a filesystem
    help       Print detailed help for the given command
    index      Create or display the index of files to back up
    on         Backup a remote machine to the local one
    restore    Extract files from a backup set
    save       Save files into a backup set (note: run "bup index" first)
    tag        Tag commits for easier access
    web        Launch a web server to examine backup sets

Other available commands:
    bloom                features             import-rsnapshot     margin               on--server           split
    cat-file             gc                   init                 memtest              prune-older          tick
    daemon               get                  join                 meta                 random               version
    damage               import-duplicity     list-idx             midx                 rm                   xstat
    drecurse             import-rdiff-backup  ls                   mux                  server

See 'bup help COMMAND' for more information on a specific command.
```

Could you confirm on your machine?

**Comment by Uatschitchun** [comment]
I'll test and report back
